include/SDL_config.h

```diff
@@ -0,0 +1,16 @@
+/** @name Frames / MSF Conversion Functions
+ *  Conversion functions from frames to Minute/Second/Frames and vice versa
+ */
+/*@{*/
+#define CD_FPS 75
+#define FRAMES_TO_MSF(f, M, S, F) \
+	{ \
+		int value = f; \
+		*(F) = value % CD_FPS; \
+		value /= CD_FPS; \
+		*(S) = value % 60; \
+		value /= 60; \
+		*(M) = value; \
+	}
+#define MSF_TO_FRAMES(M, S, F) ((M)*60 * CD_FPS + (S)*CD_FPS + (F))
+/*@}*/

```

Comment:
We do have frames_to_msf and msf_to_frames, which these could call as a simple wrapper (a bit more DRY)

Reply:
I changed FRAMES_TO_MSF to call frames_to_msf. Meanwhile, MSF_TO_FRAMES is only one line and calling msf_to_frames will make the code size bigger, so I kept as is for now.

Reply:
During the build system file integration, I had to clean up the code to make it build in different platforms. The code was moved to a different file and restored so that it will build properly.